stdlib/testing/prometheus/histogramQuantile_test.flux

```diff
@@ -0,0 +1,54 @@
+package prometheus_test
+
+import "experimental/prometheus" 
+import "testing"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,string,string,string,string,double
+#group,false,false,false,true,true,false,true,false
+#default,_result,,,,,,,
+,result,table,_time,_measurement,_field,url,le,_value
+,,0,2018-05-22T13:00:00Z,prometheus,prometheus_test_metric,http://prometheus.test,100,0.1
```

Comment:
I would have thought that the `_value` column would be all whole numbers (despite being of type `double` which is the only scalar numeric type supported by Prometheus I guess), since it represents a count... am I missing something?

Reply:
I expected the same :point_up: